app/lib/privacy_policy/src/table_of_contents/document_section_view.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:collection/collection.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+import 'package:meta/meta.dart';
+
+import '../privacy_policy_src.dart';
+
+/// View for a document section inside the table of contents.
```

Comment:
```suggestion
/// View for a document section inside the table of contents.
///
```

https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph